src/AWS.Deploy.Orchestration/Orchestrator.cs

```diff
@@ -110,21 +107,24 @@ public async Task<List<Recommendation>> GenerateRecommendationsFromSavedDeployme
                 throw new InvalidCliArgumentException(DeployToolErrorCode.DeploymentProjectPathNotFound, $"The path '{deploymentProjectPath}' does not exists on the file system. Please provide a valid deployment project path and try again.");
 
             var engine = new RecommendationEngine.RecommendationEngine(new List<string> { deploymentProjectPath }, _session);
-            var additionalReplacements = await GetReplacements();
-            return await engine.ComputeRecommendations(additionalReplacements);
+            return await engine.ComputeRecommendations();
         }
 
-        public async Task<Dictionary<string, string>> GetReplacements()
+        public async Task AddReplacementTokens(Recommendation recommendation, string cloudApplicationName)
```

Comment:
Rename method to something more appropriate.

Reply:
Fixed